docker/docker-entrypoint.sh

```diff
@@ -8,8 +8,10 @@ if [ "${1:0:1}" = '-' ]; then
 	set -- sentry "$@"
 fi
 
+VALID_COMMANDS=$(python -c 'import pkgutil; import os.path; import sentry.runner.commands as c; print("|".join([name for _, name, _ in pkgutil.iter_modules([os.path.dirname(c
```

Comment:
If we're going to do this, I'd rather not subprocess. Just check the files on disk.

`-f src/sentry/runner/commands/"$1".py` or whatever.

Reply:
Eh, I didn't wanna special-case `__init__.py` and be Pythonic. I see your point tho.

Reply:
We could easily just do something like, `if $1 does not start with _` but not sure how to do that in bash rn. It's also not _that_ big of a deal if someone did `sentry __init__` and it tried to run the command. It won't do anything, and would just exit 1. It's just a false positive with no real world collision.